src/lib/serviceRoutinesV2/getEntityAttribute.cpp

```diff
@@ -74,7 +74,8 @@ std::string getEntityAttribute
 
   // 01. Fill in QueryContextRequest
   parseDataP->qcr.res.fill(compV[2], type, "false", EntityTypeEmptyOrNotEmpty, "");
-
+  parseDataP->qcr.res.metadataList.fill(ciP->uriParam["metadata"]);
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/2590/commits/126e9506763c2389da6cac16d89e15da2a271fba